src/vs/workbench/contrib/terminal/browser/terminalTabbedView.ts

```diff
@@ -146,6 +166,18 @@ export class TerminalTabbedView extends Disposable {
 		this._height = height;
 	}
 
+	private _createButton(): void {
```

Comment:
this is how the search editor does things for the `Toggle Search Details` button